package.json

```diff
@@ -22,14 +22,14 @@
     "reactjs-popup": "^1.5.0"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "export SKIP_PREFLIGHT_CHECK=true && react-scripts start",
+    "build": "export SKIP_PREFLIGHT_CHECK=true && react-scripts build",
```

Comment:
Just curious, don't want to drag up painful memories, but what does create-react-app complain about to warrant the skip preflight check? 😂

Reply:
Haha yeah this stuff sux pretty bad haha. Create-react-app bundles a version of jest and babel-jest which don't work with testing-framework. Install the latest version of jest and babel-jest and your tests work! But then "build" and "start" don't work with the preflight check cos you're bypassing the bundled dependencies. Even though they're only dependencies for testing

Reply:
Upgrading create-react-app is also a pain